README.md

```diff
@@ -36,20 +36,31 @@ This module gives you the possibility to integrate OneSignal into you're Appcele
 ### Usage
 1. Register device for Push Notifications
    
-   ```
+   ```js
    	    // This registers your device automatically into OneSignal
        var onesignal = require('com.williamrijksen.onesignal');
    ```
 1. To add the possibility to target people for notifications, send a tag:
    
-   ```
+   ```js
        onesignal.sendTag({ key: 'foo', value: 'bar' });
-   ```
+   ```js
```

Comment:
Why? It enables syntax-highlighting (e.g. https://github.com/hansemannn/ti.googlemaps and https://github.com/hansemannn/ti.paypal)